xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -811,6 +810,9 @@ void CGUIWindowManager::ActivateWindow_Internal(int iWindowID, const std::vector
 void CGUIWindowManager::CloseDialogs(bool forceClose) const
 {
   CSingleLock lock(g_graphicsContext);
+  if (m_activeDialogs.empty())
```

Comment:
It's there to protect against the iterator debug assertion,  it doesn't like empty containers